dev/simran0117/Sample.ipynb

```diff
@@ -0,0 +1,394 @@
+       "0               7.0              0.27         0.36            20.7      0.045   \n",
+       "1               6.3              0.30         0.34             1.6      0.049   \n",
+       "2               8.1              0.28         0.40             6.9      0.050   \n",
+       "3               7.2              0.23         0.32             8.5      0.058   \n",
+       "4               7.2              0.23         0.32             8.5      0.058   \n",
+       "...             ...               ...          ...             ...        ...   \n",
+       "4893            6.2              0.21         0.29             1.6      0.039   \n",
+       "4894            6.6              0.32         0.36             8.0      0.047   \n",
+       "4895            6.5              0.24         0.19             1.2      0.041   \n",
+       "4896            5.5              0.29         0.30             1.1      0.022   \n",
+       "4897            6.0              0.21         0.38             0.8      0.020   \n",
+       "\n",
+       "      free sulfur dioxide  total sulfur dioxide  density    pH  sulphates  \\\n",
+       "0                    45.0                 170.0  1.00100  3.00       0.45   \n",
+       "1                    14.0                 132.0  0.99400  3.30       0.49   \n",
+       "2                    30.0                  97.0  0.99510  3.26       0.44   \n",
+       "3                    47.0                 186.0  0.99560  3.19       0.40   \n",
+       "4                    47.0                 186.0  0.99560  3.19       0.40   \n",
+       "...                   ...                   ...      ...   ...        ...   \n",
+       "4893                 24.0                  92.0  0.99114  3.27       0.50   \n",
+       "4894                 57.0                 168.0  0.99490  3.15       0.46   \n",
+       "4895                 30.0                 111.0  0.99254  2.99       0.46   \n",
+       "4896                 20.0                 110.0  0.98869  3.34       0.38   \n",
+       "4897                 22.0                  98.0  0.98941  3.26       0.32   \n",
+       "\n",
+       "      alcohol  quality  recommend  \n",
+       "0         8.8        6      False  \n",
+       "1         9.5        6      False  \n",
+       "2        10.1        6      False  \n",
+       "3         9.9        6      False  \n",
+       "4         9.9        6      False  \n",
+       "...       ...      ...        ...  \n",
+       "4893     11.2        6      False  \n",
+       "4894      9.6        5      False  \n",
+       "4895      9.4        6      False  \n",
+       "4896     12.8        7       True  \n",
+       "4897     11.8        6      False  \n",
+       "\n",
+       "[4898 rows x 13 columns]>"
+      ]
+     },
+     "execution_count": 53,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dataset.head"
```

Comment:
Similarly, you need to call this as a function.

Reply:
Done